web_client/package.json

```diff
@@ -3,23 +3,27 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@cogment/cogment-js-sdk": "^1.26.1",
+    "@cogment/cogment-js-sdk": "^2.0.1",
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
     "@types/google-protobuf": "^3.15.5",
     "google-protobuf": "^3.18.0-rc.2",
     "grpc-tools": "^1.11.2",
+    "jsdoc": "^3.6.10",
     "prettier": "^2.4.1",
+    "protobufjs": "git+https://github.com/protobufjs/protobuf.js.git#d13d5d5688052e366aa2e9169f50dfca376b32cf",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-scripts": "4.0.3",
     "serve": "^13.0.2",
+    "tmp": "^0.2.1",
     "typescript": "^4.4.3",
+    "uglify-js": "^3.14.5",
```

Comment:
I don't see new imports and yet there are new dependencies, why is that?

Reply:
the snapshot version of protobufjs fails to include some of its dependencies in the package.json. I don't know how they did this, but they did

Reply:
So now cogment-js-sdk-generate will install additional dependencies to make protobufjs work

Reply:
Shouldn't those dependencies part of the js sdk dependencies if they are required ?